src/txn/txn_timestamp.c

```diff
@@ -743,7 +743,7 @@ __wt_txn_set_prepare_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t prepare_
          * (There is a slight extra wrinkle at the moment, because it is possible for a transaction
          * to prepare and commit and be interacted with before it becomes durable. Currently such
          * transactions _must_ be replayed identically by the application to avoid inconsistency,
-         * or avoided. FIXME-WT-8747: remove this note when WT-8747 fixes this.)
```

Comment:
We've decided to [document this behaviour but leave it unchanged](https://jira.mongodb.org/browse/WT-8747?focusedCommentId=4376245&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4376245). As such I've left the comment in place.